scripts/CeladonGym.asm

```diff
@@ -135,23 +135,23 @@ CeladonGymText1:
 .done
 	jp TextScriptEnd
 
-CeladonGymText_48a5e:
-	text_far _CeladonGymText_48a5e
+ErikaPreBattleText:
+	text_far _ErikaPreBattleText
 	text_end
 
-CeladonGymText_48a63:
-	text_far _CeladonGymText_48a63
+ReceivedRainbowbadgeText:
```

Comment:
Maybe we're not consistent, but I think it would be better to capitalize "Badge"